### PR TITLE
add two lines to the config file to allow suppressing the SMTP authentication in PHPMailer

### PR DESCRIPTION
**Pull request by HongPong**
This is one side of the modification. The other side is in pkp-lib.

### CODE REVIEW

**Comment by HongPong** [comment]
See issue https://github.com/pkp/pkp-lib/issues/5781

File: config.TEMPLATE.inc.php [hunk 1]
```diff
@@ -317,6 +317,9 @@
 ; smtp_username = username
 ; smtp_password = password
 
+; Enable suppressing verification of SMTP certificate in PHPMailer
+: smtp_suppress_cert_check = On
```
Comment (asmecher) [inline config.TEMPLATE.inc.php:321]: Minor typo: that colon at the start of the line should be a semicolon.

File: config.TEMPLATE.inc.php [hunk 2]
```diff
@@ -317,6 +317,9 @@
 ; smtp_username = username
 ; smtp_password = password
 
+; Enable suppressing verification of SMTP certificate in PHPMailer
```
Comment (asmecher) [inline config.TEMPLATE.inc.php:320]: I'd suggest adding a note that this is not recommended as per the PHPMailer documentation (see under "Updating CA certificates" at https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting).

**Comment by HongPong** [comment]
Ahh thank you sorry i missed the correct character, good eye - 👁️ 
I added a better notification that this is not the recommended config, let me know if you think the phrase should change.

**Comment by asmecher** [comment]
Both good, thanks!

**Pull request merged**